server/container/cli/src/main/java/org/apache/james/cli/type/CmdType.java

```diff
@@ -22,44 +22,44 @@
  * Enumeration of valid command types.
  */
 public enum CmdType {
-    ADDUSER("adduser", "username","password"),

```

Comment:
Nope, it's not case sensitive.

(And the PR build is Green)

CLI is the entry point, and the first interactions you get with James. Thus, it is very important to me to get the experience as nice as possible. And doing this non-breaking change is a step in this direction.